### ISSUE TITLE
Workflow is only triggered if `.cirrus.yml` exists in the "master" branch

### ISSUE THREAD

**Issue by riccardomurri** [cirruslabs/cirrus-ci-docs#823]
## Expected Behavior

CI workflow starts if `.cirrus.yml` is created in *any* branch.  (My intention was trying out Cirrus CI by creating the file in a feature branch, and only pushing to master when everything is working.)

## Real Behavior

Cirrus CI did not even list my repository in the "Active repositories" tab in the web UI until I pushed the `.cirrus.yml` file to the "master" `(default) branch.

# Related Info

I guess it's fine to have it like this but can you please document it?

**Comment by fkorotkov** [comment]
Yeah, it's a bit buggy. Repositories appear in your Active and on the main page if there is at least one build on a default branch. You still should see build on the repository page and commit checks should be propagated to GitHub UI.

We'll change the behaviour to not require a build on a default branch. 👌
